.eslintrc.json

```diff
@@ -1,22 +1,34 @@
 {
-  "env": {
-    "browser": true,
-    "es2021": true
-  },
-  "extends": [
-    "plugin:react/recommended",
-    "airbnb",
-    "plugin:prettier/recommended"
-  ],
-  "parserOptions": {
-    "ecmaFeatures": {
-      "jsx": true
+    "env": {
+        "browser": true,
+        "es2021": true
     },
-    "ecmaVersion": "latest",
-    "sourceType": "module"
-  },
-  "plugins": ["react"],
-  "rules": {
-    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
-  }
+    "extends": [
+        "plugin:react/recommended",
+        "airbnb"
+    ],
```

Comment:
we are no longer extending any prettier dependencies in `eslintrc.json` therefore Eslint default formatting rules are the ones being used instead of using prettier for formatting 
I tested and prettier's rules only run only when you manually run `npm run format` which calls `npx pretty-quick`

Reply:
> we are no longer extending any prettier dependencies in `eslintrc.json` therefore Eslint default formatting rules are the ones being used instead of using prettier for formatting I tested and prettier's rules only run only when you manually run `npm run format` but so far `npm run lint` doesn't consider prettier

the pre-commit script is now set up but the problem stated above remains and prettier rules are not being used for formatting before committing

Reply:
Can you elaborate on that and explain the issue